src/slidem-slide-base.js

```diff
@@ -108,15 +108,16 @@ export class SlidemSlideBase extends GluonElement {
     }
   }
 
+  #steps = Array.from(this.querySelectorAll('[reveal]'));
```

Comment:
Will this work outside the connectedCallback? If I understand correctly this syntax will run the querySelector after the constructor, at which points there may not be any children yet.

Maybe it's safer to only initialize the field here and still set the value in the connectedCallback?

Reply:
yeah it might not work, worth testing